### PR TITLE
fix: adding in securityAdmin role for exp prj

### LINKED ISSUE #321: Experimentation: missing permission on client project
as an application developer, working in the experimentation project, I need to be able to set IAM permissions on the project and any resources I have deployed underneath.

### PR DESCRIPTION
**Pull request by lucstjean-ssc**
Closes #321

**Features:** 

- Allow users in the experimentation project to set IAM permissions on the project and any resources deployed underneath.
  - adding in `roles/iam.securityAdmin`

### CODE REVIEW

**Review by davelanglois-ssc — CHANGES_REQUESTED** [review]
would you mind regenerating the readme.md and also making the adjustment in this file please https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/cef7c7f4b33c0516a9259e6fad83eedb55b4868a/solutions/project/project-experimentation/securitycontrols.md#guardrail-2---global-admins-should-be-secured

**Comment by lucstjean-ssc** [comment]
> would you mind regenerating the readme.md and also making the adjustment in this file please https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/cef7c7f4b33c0516a9259e6fad83eedb55b4868a/solutions/project/project-experimentation/securitycontrols.md#guardrail-2---global-admins-should-be-secured

Both comments have been actioned.

**Review by davelanglois-ssc — APPROVED** [review]

**Review by stephanemillaire-ssc — APPROVED** [review]

**Comment by fmichaelobrien** [comment]
reviewing...

File: solutions/project/project-experimentation/README.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ This package has no sub-packages.
 
 1.  Move into the local package:
     ```shell
-    cd "./project-experimentation/"
+    cd ".//solutions/project/project-experimentation/"
```
Comment (fmichaelobrien) [inline solutions/project/project-experimentation/README.md:69]: double slash  //solutions to single /
https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/tree/main/solutions/project/project-experimentation

**Review by fmichaelobrien — APPROVED** [review]
LGTM pending the //

**Review by cartyc — APPROVED** [review]
/lgtm

File: solutions/project/project-experimentation/README.md (hunk 1 above)
Comment (lucstjean-ssc) [reply solutions/project/project-experimentation/README.md:69]: The generate-kpt-pkg-docs added this in?

Comment (fmichaelobrien) [reply solutions/project/project-experimentation/README.md:69]: works either way 
cd ../solutions/project...
cd ..//solutions/project...

**Pull request merged**